compiler/resolution/functionResolution.cpp

```diff
@@ -1706,6 +1895,9 @@ handleSymExprInExpandVarArgs(FnSymbol*  workingFn,
         }
       }
 
```

Comment:
@vasslitvinov - note that a reviewer can hide whitespaces changes when reviewing if they append `?w=l` to the url.  I personally wouldn't avoid making whitespace changes when adding code like this - it's more important to be able to read the file on its own cleanly than avoid an extra step when tracking down changes to that line.